nilearn/datasets/neurovault.py

```diff
@@ -2361,6 +2420,9 @@ def fetch_neurovault(
         counts and add it to the result. Also add to the result a
         vocabulary list. See ``sklearn.CountVectorizer`` for more info.
 
+    resample : bool, optional (default=False)
+        Resamples downloaded images to a 4x4x4 grid before saving them, to save disk space.
```

Comment:
'4x4x4 grid' : WDYM ?
The output resolution is actually 3mm x 3mm x 3mm

Reply:
thanks, probably initially used 4x4x4 and switched to 3x3x3